README.md

```diff
@@ -218,6 +219,10 @@ about 1 minute to compile times and improves the runtime by about 11%. If
 you would like to disable LTO pass `-DLINK_TIME_OPTIMIZATION=OFF` when
 generating a build.
 
+### GPU acceleration
+The default solver for initial placement is based on CPU. If you would like 
```

Comment:
Suggest the following rewordings

The default solver for initial placement is based on CPU -> The default solver for initial placement is single threaded.

set `-DGPU=true` at the compiling time -> set `-DGPU=true` at compile time.